### PR TITLE
Fix/hardcoded endpoint

### PR DESCRIPTION
**Pull request by 0xBEEFCAF3**
* Some formatting
* rename loin -> nolooking
* added new config var `endpoint`. Open to other name suggestions? maybe P2EP?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,9 +77,9 @@ In other words, your grandmother will be able to somewhat privately open a bunch
    ```
 
 5. `cargo run --features=test_paths -- --conf CONFIGURATION_FILE_PATH FEE_RATE DEST_NODE_URI AMOUNT_IN_SATS [DEST_NODE_URI AMOUNT_IN_SATS ...] [CHAIN_WALLET_AMOUNT_SATS]`
-6. Copy BIP21 from command line output and paste it into one of the supported wallets. I use [the payjoin crate client](https://github.com/Kixunil/payjoin/tree/master/payjoin-client) to make a payjoin right from regtest bitcoind.
-7. Confirm the transaction and move some sats over the new channel
-
+6. Copy your lnd tls.cert & admin.macaroon to root dir
```
[the remark this answers is not in the document]
Comment (0xBEEFCAF3) [reply README.md:80]: I want to indicate that this a step. I agree, prefer the end sysadmin doesn't copy paste secrets. in that case I think we should make two new config vars for paths to the secrets

File: static/index.html [hunk 2]
```diff
@@ -1,63 +1,70 @@
...
+    <div id="send" class="invisible">
+      <a href="" id="bip21"></a>
+    </div>
+    <script type="text/javascript">
+      function add_channel() {
+        var table = document.getElementById("channels");
+        table.innerHTML =
+          table.innerHTML +
+          '<tr><td><input type="text"></td><td><input type="text"></td></tr>';
+      }
```
[the remark this answers is not in the document]
Comment (0xBEEFCAF3) [reply static/index.html:36]: This is just formatting ( I mentioned in the PR description). If you insist to have its own commit I was going to do a Frontend revamp anyway. Happy to move it there. But I don't think its a huge deal. its just removing extraneous tabs

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (0xBEEFCAF3) [reply README.md:80]: In that case, I think those extra vars should be in their own PR. Thoughts?

[the remark this answers is not in the document]
Comment (DanGould) [reply README.md:80]: There are already config vars for paths to secrets. I don't think this step needs to change

https://github.com/chaincase-app/nolooking/blob/f266ccd7c59d4e2cac05bf0cd6226c9c28c81ad7/README.md?plain=1#L70-L77

https://github.com/chaincase-app/nolooking/blob/f266ccd7c59d4e2cac05bf0cd6226c9c28c81ad7/config_spec.toml#L23-L33

[the remark this answers is not in the document]
Comment (0xBEEFCAF3) [reply README.md]: Ah good point. Totally missed that somehow

**Comment by DanGould** [comment]
Please rebase so the history is git bisect-able https://github.com/rust-bitcoin/rust-bitcoin/blob/master/CONTRIBUTING.md#contribution-workflow

**Pull request closed without merging**